.buildkite/pipeline.release-stable.yml

```diff
@@ -31,8 +31,18 @@ steps:
     command: ".buildkite/steps/publish-debian-package.sh"
     env:
       CODENAME: "stable"
+      DEB_S3_BUCKET: "apt.buildkite.com/buildkite-agent"
     agents:
-      queue: "deploy-legacy"
+      queue: "deploy"
+    plugins:
+      - ecr#v2.0.0:
+          login: true
+          account-ids: "032379705303"
+      - docker#v3.5.0:
+          image: "032379705303.dkr.ecr.us-east-1.amazonaws.com/deploytools:2020.03"
+          propagate-environment: true
+          tmpfs:
```

Comment:
:+1: 

I wonder if there's any way to validate that the GPG dir is not persisting to disk prior to downloading the secret key. Otherwise it feels like a slightly fragile security model, with this tmpfs specification so far away from the key being placed there. 

```
pda@x1pda ~ $ docker run --rm -it --tmpfs /hello archlinux mount | grep hello
tmpfs on /hello type tmpfs (rw,nosuid,nodev,noexec,relatime)
```